CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing Guildelines for the L.E.O WEBSITE
+
+* * *
+
+There are mainly three sectors in which anyone can contribute to the our Website:
+* Authoring a post (The Blog could be about anything realted to **Literature**.)
+* Reporting an issue and giving suggestions.
+* Localizing it or contributing to the code.
+
+---
+
+## Points to remember:
+### 1. Local Setup:
+    
+   For setting up locally Fork the repositry and clone it using the commands given below in the terminal.
+    
+ ```sh
+    # create a local clone of the repository
+    git clone https://github.com/YOUR-USERNAME/leomvit.github.io.git
+    # switch to the created directory
+    cd leomvit.github.io.git
+    # start local server
+    bundle exec jekyll serve
+ ```
+  This will start the Jekyll Server.
```

Comment:
Add instructions for setting up the repo in Docker
cc @cseas

Reply:
Why do we need Docker?